src/utils/nodemailer.js

```diff
@@ -0,0 +1,35 @@
+// create a nodemailer transport config and function to send emails
+
+const nodemailer = require('nodemailer');
+
+const transporter = nodemailer.createTransport({
+    host: "smtp-mail.outlook.com", // hostname
+    secureConnection: false, // TLS requires secureConnection to be false
+    port: 587, // port for secure SMTP
+    tls: {
+       ciphers:'SSLv3'
+    },
+    auth: {
+        user: process.env.EMAIL,
+        pass: process.env.EMAIL_PASSWORD,
+    }
```

Comment:
It's configured for using our MDS email? :)

Reply:
yes ! outlook protocol for mds emails, you just need to put your credentials in .env